docs/WELCOME_TUTORIAL.markdown

```diff
@@ -58,7 +58,7 @@ In case you missed it, the README has [VERY Quick Start instructions that show y
 
 Clone down the reference application provided by the Grove project. For this, you will need to have a command-line terminal and [git](https://git-scm.com/downloads) for version-control.
 
-    git clone --recursive ssh://git@project.marklogic.com:7999/nacw/grove-react-template.git
+    git clone --recursive https://github.com/marklogic-community/grove-react-template.git
     cd grove-react-template
```

Comment:
Not seeing the context. Is this to clone the react template for contributing to it? If not, we should really recommend using grove-cli instead.

Reply:
It was decided to remove this file completely due to it being mostly out-of-date now.